src/main/java/io/cdap/e2e/pages/locators/CdfBigQueryPropertiesLocators.java

```diff
@@ -60,4 +63,32 @@
 
   @FindBy(how = How.XPATH, using = "//*[@data-cy=\"cmekKey\" and @class='MuiInputBase-input']")
   public static WebElement cmekKey;
+
+  @FindBy(how = How.XPATH, using = "//*[@data-cy='partitionFrom' and @class='MuiInputBase-input']")
+  public static WebElement partitionStartDate;
```

Comment:
This must not live here. The whole file should reside in google-cloud.

Reply:
BigQuery is being used as Sink across different plugin testcases. thats why it is placed in e2e.
@ksahilCF